inbc-program/inbc/parser/parser.py

```diff
@@ -73,10 +73,10 @@ def parse_source_args(self) -> None:
                                     type=lambda x: validate_string_less_than_n_characters(
                                         x, 'str', 200),
                                     help='Name to store the GPG key information')
-        app_add_parser.add_argument('-source', '--s', required=True,
```

Comment:
common practice with argparse (and with most Linux CLI tools) is to use two dashes for long arguments and one dash for short arguments. see for example https://docs.python.org/3/howto/argparse.html

Reply:
(I looked at the AOTA parser and it follows this - for short and -- for long args convention as well)